Application/EdFi.Ods.Api.IntegrationTestHarness/EdFi.Ods.Api.IntegrationTestHarness.csproj

```diff
@@ -50,7 +51,6 @@
     <ProjectReference Include="..\..\..\Ed-Fi-ODS\Application\EdFi.Ods.Composites.Enrollment\EdFi.Ods.Composites.Enrollment.csproj" />
     <ProjectReference Include="..\..\..\Ed-Fi-ODS\Application\EdFi.Ods.Composites.Test\EdFi.Ods.Composites.Test.csproj" />
     <ProjectReference Include="..\..\..\Ed-Fi-ODS\Application\EdFi.Ods.Features\EdFi.Ods.Features.csproj" />
-    <ProjectReference Include="..\..\..\Ed-Fi-ODS\Application\EdFi.Ods.Profiles.Test\EdFi.Ods.Profiles.Test.csproj" />
```

Comment:
Remove the project folder as per the ticket acceptance criteria

Reply:
@simpat-jesus I think it is fine to keep the xml if it helps to load it to Test Admin database. (either in the project or as embeded resource in the test harness. We do want to make sure that test harness is testing the profile defined in the database.

Reply:
@vimayya aceptance criteria was edited so I think Its ok and I will resolve this conversation